pages/api/dashboard/chart.js

```diff
@@ -0,0 +1,51 @@
+import prisma from "../../../lib/prisma";
+
+export default async (req, res) => {
+  const reqBody = JSON.parse(req.body);
+
+  if (req.method === "POST") {
+    try {
```

Comment:
Try creating single middleware to handle Errors and remove try catch from each api